core/src/main/java/io/lionweb/lioncore/java/language/Enumeration.java

```diff
@@ -1,12 +1,17 @@
 package io.lionweb.lioncore.java.language;
 
+import io.lionweb.lioncore.java.LionWebVersion;
 import io.lionweb.lioncore.java.self.LionCore;
 import java.util.List;
 import java.util.Objects;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
 public class Enumeration extends DataType<Enumeration> implements NamespaceProvider {
```

Comment:
`NamespaceProvider` is not an entity in the M3 anymore. It might be convenient here, but it's also confusing. Also: can we get rid of TODOs that mention it?

Reply:
It turns out that the TODO messages are outdated as all the containers we use implement NamespaceProvider, so I removed all such comments